packages/remark-lint-no-shell-dollars/readme.md

```diff
@@ -31,13 +31,11 @@ This rule is included in the following presets:
 ````markdown
 ```sh
 echo a
-echo a > file
```

Comment:
these example should still work, why were the removed?

Reply:
Thank you, specs fixed: deletions reverted, new ones have been added.